website/content/docs/concepts/filtering/index.mdx

```diff
@@ -0,0 +1,113 @@
+---
+layout: docs
+page_title: Filtering
+description: |-
+  An introduction to the filtering syntax used in Vault.
+---
+
+# Filter expressions
+
+Filter expressions can be used by various parts of Vault to provide useful
+functionality. This page describes the overall syntax; see the sub-pages for
+information on the specific values available for filtering for various features,
+including examples.
+
+Each filter expression has matching operators composed with selectors and
+values.
+
+## Create expressions
+
+A single expression is a matching operator with a selector and value. They are
+written in plain text format, and Boolean logic and parenthesization are
+supported. In general whitespace is ignored, except within literal strings.
+
+### Matching operators
```

Comment:
Would it be possible to add some examples throughout, or maybe a big example at the end that puts all these things together?

Reply:
This page has been taken from Boundary in order for Vault to not have to reference their page in our docs.

https://developer.hashicorp.com/boundary/docs/concepts/filtering